x-pack/metricbeat/module/sql/query/_meta/data_postgres_variables.json

```diff
@@ -10,267 +10,349 @@
-                "gin_fuzzy_search_limit": 0,
-                "gin_pending_list_limit": 4096,
-                "join_collapse_limit": 8,
-                "lock_timeout": 0,
-                "log_autovacuum_min_duration": -1,
-                "log_file_mode": 600,
-                "log_min_duration_statement": -1,
-                "log_rotation_age": 1440,
-                "log_rotation_size": 10240,
-                "log_temp_files": -1,
-                "maintenance_work_mem": 65536,
-                "max_connections": 100,
-                "max_files_per_process": 1000,
-                "max_function_args": 100,
-                "max_identifier_length": 63,
-                "max_index_keys": 32,
-                "max_locks_per_transaction": 64,
-                "max_pred_locks_per_transaction": 64,
-                "max_prepared_transactions": 0,
-                "max_replication_slots": 0,
-                "max_stack_depth": 2048,
-                "max_standby_archive_delay": 30000,
-                "max_standby_streaming_delay": 30000,
-                "max_wal_senders": 0,
-                "max_wal_size": 64,
-                "max_worker_processes": 8,
-                "min_wal_size": 5,
-                "pg_stat_statements.max": 10000,
-                "port": 5432,
-                "post_auth_delay": 0,
-                "pre_auth_delay": 0,
-                "random_page_cost": 4,
-                "segment_size": 131072,
-                "seq_page_cost": 1,
-                "server_version_num": 90503,
-                "shared_buffers": 16384,
-                "statement_timeout": 0,
-                "superuser_reserved_connections": 3,
-                "tcp_keepalives_count": 9,
-                "tcp_keepalives_idle": 7200,
-                "tcp_keepalives_interval": 75,
-                "temp_buffers": 1024,
-                "temp_file_limit": -1,
-                "track_activity_query_size": 1024,
-                "unix_socket_permissions": 777,
-                "vacuum_cost_delay": 0,
-                "vacuum_cost_limit": 200,
-                "vacuum_cost_page_dirty": 20,
-                "vacuum_cost_page_hit": 1,
-                "vacuum_cost_page_miss": 10,
-                "vacuum_defer_cleanup_age": 0,
-                "vacuum_freeze_min_age": 50000000,
-                "vacuum_freeze_table_age": 150000000,
-                "vacuum_multixact_freeze_min_age": 5000000,
-                "vacuum_multixact_freeze_table_age": 150000000,
-                "wal_block_size": 8192,
-                "wal_buffers": 512,
-                "wal_keep_segments": 0,
-                "wal_receiver_status_interval": 10,
-                "wal_receiver_timeout": 60000,
-                "wal_retrieve_retry_interval": 5000,
-                "wal_segment_size": 2048,
-                "wal_sender_timeout": 60000,
-                "wal_writer_delay": 200,
-                "work_mem": 4096
+            "allow_system_table_mods": "off",
```

Comment:
Smaller example here too, please.

Reply:
This file is only used by integration test for now and not in the external doc, so leaving as-is.